tests/libcxx/expected_results.txt

```diff
@@ -654,6 +654,12 @@ std/language.support/support.limits/support.limits.general/chrono.version.pass.c
 # "should not be defined when TEST_HAS_BUILTIN(__builtin_addressof) || TEST_GCC_VER >= 700 is not defined!"
 std/language.support/support.limits/support.limits.general/memory.version.pass.cpp FAIL
 
+# libc++ doesn't implement P2186R2 Removing Garbage Collection Support  for now
+# Recheck after LLVM update. The tests might be removed in the future by upstream
+# because libc++ wants to implement P2186R2 in all standard modes.

```

Comment:
The test runner will inform us in the future if the test goes away or starts unexpectedly passing, so we don't really need the additional explanation here:
```suggestion
# libc++ doesn't implement P2186R2 Removing Garbage Collection Support
```